omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,103 @@
+Complete workflow for hosting data for a publication
+====================================================
+
+The following describes the steps of a complete workflow for using OMERO to
+host public data associated with a publication. It is illustrated using an
+example publication from our lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_.
+
+Group setup
+-----------
+
+#. Create a dedicated read-only group to host the raw data underlying the
+   publication (see :doc:`cli/usergroup`)
+#. Add all the authors of the paper to this new group.
+#. If you have not already done so, configure the :ref:`public_user`.
```

Comment:
This line should probably be moved under "Configuring OMERO.web", especially since you may want to setup a new OMERO.web before doing this.

Reply:
Maybe move the whole Configuring OMERO.web section to the top, above Group setup, since you will not be "making all the data publicly available" until you have done the web setup.